Traceback (most recent call last):
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 1488, in __call__
    return self.wsgi_app(environ, start_response)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 1466, in wsgi_app
    response = self.handle_exception(e)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\GEAM\app\views\clienti.py", line 114, in index
    return render_template('clienti/index.html',
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 150, in render_template
    return _render(app, template, context)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 131, in _render
    rv = template.render(context)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\jinja2\environment.py", line 1301, in render
    self.environment.handle_exception()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\GEAM\app\templates\clienti\index.html", line 2, in top-level template code
    {% extends "base.html" %}
  File "C:\GEAM\app\templates\base.html", line 220, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^
  File "C:\GEAM\app\templates\clienti\index.html", line 100, in block 'content'
    {% include 'clienti/partials/lista_clienti.html' with context %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
           ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: clienti/partials/lista_clienti.html
Traceback (most recent call last):
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 1488, in __call__
    return self.wsgi_app(environ, start_response)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 1466, in wsgi_app
    response = self.handle_exception(e)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\GEAM\app\views\clienti.py", line 114, in index
    return render_template('clienti/index.html',
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 150, in render_template
    return _render(app, template, context)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 131, in _render
    rv = template.render(context)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\jinja2\environment.py", line 1301, in render
    self.environment.handle_exception()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\GEAM\app\templates\clienti\index.html", line 2, in top-level template code
    {% extends "base.html" %}
  File "C:\GEAM\app\templates\base.html", line 220, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^
  File "C:\GEAM\app\templates\clienti\index.html", line 100, in block 'content'
    {% include 'clienti/partials/lista_clienti.html' with context %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
           ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: clienti/partials/lista_clienti.html
Traceback (most recent call last):
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 1488, in __call__
    return self.wsgi_app(environ, start_response)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 1466, in wsgi_app
    response = self.handle_exception(e)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\GEAM\app\views\clienti.py", line 114, in index
    return render_template('clienti/index.html',
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 150, in render_template
    return _render(app, template, context)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 131, in _render
    rv = template.render(context)
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\jinja2\environment.py", line 1301, in render
    self.environment.handle_exception()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\GEAM\app\templates\clienti\index.html", line 2, in top-level template code
    {% extends "base.html" %}
  File "C:\GEAM\app\templates\base.html", line 220, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^
  File "C:\GEAM\app\templates\clienti\index.html", line 100, in block 'content'
    {% include 'clienti/partials/lista_clienti.html' with context %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
           ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cristiano.roffilli\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: clienti/partials/lista_clienti.html
